cypress/e2e/eda/Rulebook-Activations/rulebook-activations-crud.cy.ts

```diff
@@ -136,13 +136,15 @@ describe('EDA rulebook activations- Edit, Delete', () => {
   it('can enable and disable a Rulebook Activation', () => {
     cy.navigateTo(/^Rulebook Activations$/);
     cy.filterTableByText(edaRBA.name);
+    cy.wait(10000);
```

Comment:
10s seems like a long wait here. What's the reason for the hardcoded wait? Can we avoid it?

Reply:
@vidyanambiar - this delay was here up until 2 days ago. This test was failing now,  so I just reversed the changes made 2 days ago that removed this delay to see if this is the reason for the failure. I agree the delay is long - was planning to look at some other options.